src/test/java/rx/internal/operators/OperatorObserveOnTest.java

```diff
@@ -834,4 +834,16 @@ public void testErrorDelayedAsync() {
         ts.assertError(TestException.class);
         ts.assertNotCompleted();
     }
+    
+    @Test
+    public void requestExactCompletesImmediately() {
```

Comment:
Yeah, should request 0 upfront and wait till the range completes before requesting 10. I'll change this to use the TestScheduler.